chore: push a fresh yarn.lock

abumalick opened the pull request:
### Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.


I removed yarn.lock and generated a new one.

### Fixes

When trying to run `yarn start` with last versions of node (I tried with v12.18.2 and v14.5.0) I get this error:
```
ERROR in ./demo/app.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main resolved in /path/to/repo/nuka-carousel/node_modules/@babel/helper-compilation-targets/package.json
    at applyExports (internal/modules/cjs/loader.js:491:9)
    at resolveExports (internal/modules/cjs/loader.js:507:23)
    at Function.Module._findPath (internal/modules/cjs/loader.js:635:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:953:27)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/path/to/repo/nuka-carousel/node_modules/@babel/preset-env/lib/debug.js:8:33)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
 @ multi ./demo/app.js main[0]
```

When investigating, I found out this issue in babel repository: https://github.com/babel/babel/issues/11216 where they advice to upgrade all @babel deps to v7.8.7 or above. Some developers in the issue advice to remove `yarn.lock` or `package-lock.json` and regenerate one. So I tried this last one and everything seems to be working now


#### Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

To reproduce the bug before the changes of this PR:

- Switch to last version of node LTS 14.5.0 or v12.18.2
- remove node_modules
- `yarn`
- `yarn start`

### Checklist: (Feel free to delete this section upon completion)

- [x] My code follows the style guidelines of this project (I have run `yarn lint`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes (I have run `yarn test` and `yarn test-e2e`)
- [x] My changes generate no new warnings

JoviDeCroock commented:
I think bumping them in the package.json and then running yarn again is better. Now the ones in our package.json still indicate ^older-version let’s update that.

abumalick commented:
ok for package.json

What about reseting the yarn.lock, is it ok for you or you want me to remove that?

abumalick commented:
here you go @JoviDeCroock 
- I removed my changes to yarn.lock
- I updated the versions of babel packages in package.json instead
- removed babel-core https://stackoverflow.com/a/53395718/1673761
- Run start, build and all tests
